frontend/packages/pipelines-plugin/locales/zh/pipelines-plugin.json

```diff
@@ -12,77 +14,77 @@
   "Namespace": "命名空间",
   "Type": "类型",
   "Created": "创建的",
-  "Pipeline Run Details": "管道运行详情",
+  "Pipeline run details": "管道运行详情",
   "Status": "状态",
   "Pipeline": "管道",
-  "No Task Runs Found": "未找到任务运行",
-  "Download All Task Logs": "下载所有任务日志",
-  "No Logs Found": "未找到日志",
+  "No task runs found": "未找到任务运行",
+  "Download all task logs": "下载所有任务日志",
+  "No logs found": "未找到日志",
   "Triggered by": "触发者",
-  "Task Status": "任务状态",
-  "Started": "开始的",
+  "Task status": "任务状态",
+  "Started": "开始",
   "Duration": "持续时间",
   "Unknown failure condition": "未知故障状况",
   "Failure on task {{taskName}} - check logs for details.": "任务 {{taskName}} 失败 - 检查日志详情。",
-  "Error downloading logs.": "下载日志时出错。",
+  "Error downloading logs.": "下载日志出错。",
   "An error occurred while retrieving the requested logs.": "在获取请求的日志时发生错误。",
   "Unknown error retrieving logs": "获取日志的未知错误",
   "Download": "下载",
-  "Download All": "下载所有",
+  "Download all": "下载所有",
   "Collapse": "折叠",
   "Expand": "扩展",
   "Failure - check logs for details.": "失败 - 查看日志详情。",
   "Message": "消息",
-  "Log Snippet": "日志片断",
+  "Log snippet": "日志片断",
   "Task Runs": "任务运行",
```

Comment:
Should this be `Task runs`

Reply:
It seems like this a defined model name.